PredictionIOEngine/src/main/scala/package.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright ActionML, LLC under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.routeme
+
+import grizzled.slf4j.Logger
+import scala.collection.JavaConversions._
+import org.apache.mahout.sparkbindings.SparkDistributedContext
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/heshamhamdymassoud/RouteMe-API/file/2583741693/issues/source?bid=3287637&fileBranchId=3303680#l22)